lib/active_fedora/file.rb

```diff
@@ -63,7 +63,15 @@ def ldp_connection
     # If this file has a parent with ldp#contains, we know it is not new.
     # By tracking exists we prevent an unnecessary HEAD request.
     def new_record?
-      !@exists && ldp_source.new?
+      !@exists && ldp_new_record?
```

Comment:
I'm not sure we need `==` to use `new_record?`. I'm working up a new PR to remove that. Please stand by.